doc/primitives/matmul.md

```diff
@@ -188,6 +189,7 @@ memory buffer that shares its shape with the destination buffer).
      * Runtime dimensions.
      * Three and higher dimensional matrices.
    - The layout of dropout mask has to be exactly the same as that of dst.
+   - f64 is only supported on Intel(R) Data Center GPU Max Series.
```

Comment:
There might be other platforms supporting f64 in the future, it would be better not to mention this specific platform and align with the description in [dev_guide_data_types.html](https://oneapi-src.github.io/oneDNN/dev_guide_data_types.html#intel-r-processor-graphics-and-xe-architecture-graphics) might be better:

> f64 configurations are only supported on GPU engines with HW capability for double-precision floating-point.